pages/transfer.vue

```diff
@@ -245,8 +250,9 @@ export default class Transfer extends mixins(
   }
 
   get balance(): string {
-    return this.$store.getters.getAuthBalance
+    return this.identityStore.getAuthBalance(this.urlPrefix)
```

Comment:
will it be more convenient to use if urPrefix was read internally inside getAuthBalance?

Reply:
removed, was thinking of accesing bsx balance while on rmrk chain (so we would show global balance) but this can be done next